go/chat/convsource.go

```diff
@@ -383,14 +384,26 @@ func (s *HybridConversationSource) identifyTLF(ctx context.Context, conv chat1.C
 				return nil
 			}
 
+			tlfName := msg.Valid().ClientHeader.TLFNameExpanded(conv.Metadata.FinalizeInfo)
+
 			switch conv.GetMembersType() {
 			case chat1.ConversationMembersType_TEAM:
 				// early out of team convs
 				return nil
-			default:
+			case chat1.ConversationMembersType_IMPTEAM:
+				s.Debug(ctx, "identifyTLF: implicit team TLF, looking up display name for %s", tlfName)
+
+				team, err := teams.Load(ctx, s.G().ExternalG(), keybase1.LoadTeamArg{Name: tlfName})
```

Comment:
I think it is better to load by team ID right? I think any team load by name is a round trip.